src/tasks/AppleAppBuilder/Xcode.cs

```diff
@@ -330,8 +340,30 @@ public void CreateXcodeProject(string projectName, string cmakeDirectoryPath)
 
         if (isLibraryMode)
         {
-            string dylibName = $"lib{projectName}.dylib";
-            cmakeLists = cmakeLists.Replace("%DYLIB_PATH%", $"{Path.Combine(binDir, dylibName)}");
+            string libraryPath;
+            // TODO: unify MonoAOT and NativeAOT library paths
+            // Current differences:
+            // - NativeAOT produces {ProjectName}.dylib, while MonoAOT produces lib{ProjectName}.dylib
+            // - NativeAOT places the library in the 'workspace' location ie 'publish' folder, while MonoAOT places it in 'binDir' ie 'AppBundle'
```

Comment:
A .dylib isn't usable for the end developer, you should probably produce a .framework instead.

Reply:
Thank you, this meant to be an "initial" support for the library mode. We will definitely consider going with `.framework` approach.

Reply:
I have opened https://github.com/dotnet/runtime/issues/96664 to track this.